aws-ecs/10-vpc.tf

```
@@ -6,7 +6,7 @@ module "vpc" {
   source  = "terraform-aws-modules/vpc/aws"
   version = "3.19.0"
 
-  name             = "pvault-vpc"
+  name             = "${var.deployment_id}-vpc"
```

I'm not sure what is expected of `deployment_id` to be. Never used this convention. I think that (in order to keep some values clear from users mistake) it should be something like this. That's ok if you don't agree :)
```suggestion
  name             = "${var.deployment_id}-pvault"
```

I also removed the `vpc` suffix because it's already a VPC, so it's not adding any new information.

---

done